custom/conf/app.example.ini

```diff
@@ -587,10 +587,11 @@ ENABLE_REVERSE_PROXY_AUTO_REGISTRATION = false
 ENABLE_REVERSE_PROXY_EMAIL = false
 ; Enable captcha validation for registration
 ENABLE_CAPTCHA = false
-; Type of captcha you want to use. Options: image, recaptcha
+; Type of captcha you want to use. Options: image, recaptcha, hcaptcha
```

Comment:
Tiny nit, can you add this to docs as well? Plus be more explicity about recaptcha key use for hcaptcha (perhaps even split it out into a seperate option named hcaptha...)

Reply:
Done. :slightly_smiling_face: